linters/ruff/plugin.yaml

```diff
@@ -0,0 +1,26 @@
+version: 0.1
+lint:
+  definitions:
+    - name: ruff
+      files: [python]
+      commands:
+        - name: lint
+          run: ruff check --format json ${target}
+          output: sarif
+          parser:
+            runtime: python
+            run: ${cwd}/ruff_to_sarif.py
+          batch: true
+          success_codes: [0, 1]
+      runtime: python
+      package: ruff
+      direct_configs: [pyproject.toml]
+      affects_cache:
+        - setup.cfg
+        - tox.ini
+      known_good_version: 0.0.243
+      is_recommended: false
+      good_without_config: true
```

Comment:
I thought you mentioned there was a default config you wanted to provide. Is that not the case?

Reply:
As far as I can tell, it's not necessary. [Ruff will use a default config if a pyproject.toml isn't specified](https://github.com/charliermarsh/ruff#configuration).

Reply:
Ok. Sounds good then.